third_party/murmur3/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.0)
+# project(bustub_murmur3 CXX C)
+
+# set(CMAKE_CXX_VISIBILITY_PRESET hidden)
+
+add_library(bustub_murmur3 SHARED MurmurHash3.cpp)
+
+# install(TARGETS bustub_murmur3
```

Comment:
More dead code here.  We will want to resolve this before merging anything.  I think checking things with the reference solutions will help inform what is needed.  Running things on bustub public without real implementation likely means that the murmur3 library is never being used (since there are no real backend implementations).